<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sinottico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="main_window_stack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1921</width>
      <height>901</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>410</y>
        <width>461</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="pass_label">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PASSWORD</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_label">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>USERNAME</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_edit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>200</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_edit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>290</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="project">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1921</width>
        <height>901</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="project_layout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>899</y>
      <width>1931</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="keyboard_position"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
